container-images/tcib/base/base.yaml

```diff
@@ -71,7 +71,6 @@ tcib_packages:
   - socat
   - sudo
   - tar
-  - tcib
```

Comment:
We can't remove this unless we use host files (either by volume mounts or tcib_gather_files) which is not what RelDel wants. We should package tcib first before testing it in CI I guess.